src/identity.cc

```diff
@@ -99,6 +101,10 @@ class ModelState : public BackendModel {
   // Delay time and multiplier to introduce into execution, in milliseconds.
   int execute_delay_ms_;
   int delay_multiplier_;
+
+  // output metadata to use, if an input is marked optional and not provided
```

Comment:
Do you mean input metadata here? Technically you store input metadata. It just so happens for this identity model that it is the same for the output.